VideoLocker/src/main/java/org/edx/mobile/module/notification/EdxLocalParseChannel.java

```diff
@@ -0,0 +1,45 @@
+package org.edx.mobile.module.notification;
+
+import java.io.Serializable;
+
+/**
+ * need a place to hold the logic of mapping courseId to channelId.
+ */
+public class EdxLocalParseChannel implements Serializable{
```

Comment:
@aleffert  because we also import Parse libraries. it has classes like Parse, ParsePush, ...